code/modules/tgui/tgui_panel/tgui_panel.dm

```diff
@@ -13,9 +13,9 @@
 	var/broken = FALSE
 	var/initialized_at
 
-/datum/tgui_panel/New(client/client)
+/datum/tgui_panel/New(client/client, id)
```

Comment:
do we want a default value for "id" here?

Reply:
No, but we do want to prevent making panels without an ID, i'll add safeties